Capabilitystatements/capabilitystatement-client-gp-patientdata.xml

```diff
@@ -147,6 +151,10 @@
                 <name value="patient"/>
                 <type value="reference"/>
             </searchParam>
+            <searchParam>
+                <name value="category"/>
+                <type value="token"/>
+            </searchParam>
```

Comment:
En periodofuse?

Reply:
Periodofuse zou dan bij MedicationAgreement komen? Het wordt nu niet als searchparameter genoemd in het TO, maar wel dat het gebruikt zou kunnen worden inderdaad. Toevoegen aan MedicationAgreement en AllergyIntolerance?

Reply:
Ik heb scheefgekeken, ik dacht dat dit MedicationAgreement was. Evengoed: periodofuse wordt daar wel genoemd in het TO toch?

Reply:
Excuus, je hebt helemaal gelijk. Hier zat ik scheef te kijken